src/openrct2-ui/Ui.cpp

```diff
@@ -64,3 +64,10 @@ int main(int argc, char * * argv)
     }
     return gExitCode;
 }
+
+extern "C" {
+
+int SDL_main(int argc, char *argv[]) {
+    return main(argc, argv);
+}
```

Comment:
Yes. SDL does some magic to rename the main function, but I had to disable that to get it to build as a static library iirc.